product/client/src/routes/user/auth/+page.server.ts

```diff
@@ -0,0 +1,83 @@
+import { error } from '@sveltejs/kit';
+import type { Load} from "@sveltejs/kit"
+
+import { PrismaClient, type movies, type people, type ratings } from '@prisma/client'
+import type { IJwtData, ITokenResponse, IUserResponse } from 'src/routes/common/types';
+import { signData } from '../../common/jwt';
+
+const prisma = new PrismaClient()
+
+/** @type {import('./$types').PageServerLoad} */
+export const load: Load = async ({ params, url }) => {
+    const code = url.searchParams.get("code")
+    const secret = process.env.GITHUB_OAUTH_SECRET
+    const client_id = process.env.GITHUB_CLIENT_ID
+    const accessSigningKey = process.env.JWT_ACCESS_TOKEN_SECRET_STRING
+
+    if(!secret) throw error(500, "Internal error. Code 1")
+    if(!client_id) throw error(500, "Internal error. Code 2")
+    if(!code) throw error(404, "Code not found in the request! Code 3");
+    if(!accessSigningKey) throw error(500, "Internal error! Code 4");
+
+    const admins = ["me@ddlele.com"]
```

Comment:
secrets?

Reply:
Not sure what is secret in my email :D

Reply:
Maybe it shouldn't be hardcoded, but should be fine for now.

Reply:
yeah, just thought that maybe you didnt want to have your email hardcoded / exposed here...
it's up to you :)